src/test/java/org/jabref/model/database/BibDatabaseContextTest.java

```diff
@@ -0,0 +1,68 @@
+package org.jabref.model.database;
+
+import java.io.File;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.jabref.model.metadata.FileDirectoryPreferences;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertTrue;
+
+public class BibDatabaseContextTest {
+
+    private String currentWorkingDir;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    FileDirectoryPreferences preferences;
+
+    @Before
+    public void setUp() {
+        Map<String, String> mapFieldDirs = new HashMap<>();
+        preferences = new FileDirectoryPreferences("saulius", mapFieldDirs, true);
+        currentWorkingDir = Paths.get(System.getProperty("user.dir")).toString();
```

Comment:
You can influence the fileDirectoryPreferences by mocking them, otherwise you always depend on the the settings of the user and can't have a reliable value to test.
In the CleanupTests for Renaming/Moving Files I mocked the relevant options like this:
`  fileDirPrefs = mock(FileDirectoryPreferences.class);
        when(fileDirPrefs.isBibLocationAsPrimary()).thenReturn(false); `

Have a look at `MoveFilesCleanupTest `